test/controllers/comment_controller_test.exs

```diff
@@ -34,149 +32,70 @@ defmodule CodeCorps.CommentControllerTest do
   describe "show" do
     test "shows chosen resource", %{conn: conn} do
       comment = insert(:comment)
-
-      path = conn |> comment_path(:show, comment)
-      conn = conn |> get(path)
-
-      data = json_response(conn, 200)["data"]
-
-      assert data["id"] == "#{comment.id}"
-      assert data["type"] == "comment"
-      assert data["attributes"]["body"] == comment.body
-      assert data["attributes"]["markdown"] == comment.markdown
-      assert data["relationships"]["user"]["data"]["id"] == "#{comment.user_id}"
-      assert data["relationships"]["task"]["data"]["id"] == "#{comment.task_id}"
+      conn
+      |> request_show(comment)
+      |> json_response(200)
+      |> Map.get("data")
+      |> assert_result_id(comment.id)
     end
 
     test "renders 404 when id is nonexistent", %{conn: conn} do
-      path = conn |> comment_path(:show, -1)
```

Comment:
@mackenziehicks it's a good way of saying "This `id` absolutely does not exist in the database".